fastlane/lib/fastlane/actions/download_dsyms.rb

```diff
@@ -19,13 +19,17 @@ def self.run(params)
         app.all_build_train_numbers.each do |train_number|
           app.all_builds_for_train(train: train_number).each do |build|
             download_url = build.details.dsym_url
-            result = Net::HTTP.get(URI(download_url))
-            file_name = "#{app.bundle_id}-#{train_number}-#{build.build_version}.dSYM.zip"
-            File.write(file_name, result)
-            UI.success("🔑 Successfully downloaded dSYM file for #{train_number} - #{build.build_version} to '#{file_name}'")
+            if download_url =~ URI.regexp
+              result = Net::HTTP.get(URI(download_url))
```

Comment:
Very good point, thanks for raising that. I wasn't aware this doesn't support https URLs.